src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking_FlowAnalysis.cs

```diff
@@ -1955,19 +1955,19 @@ class C
 {
     static void F1(A<object>? x1, B<object?> y1)
     {
-        (x1 ?? y1)/*T:B<object?>!*/.F.ToString();
+        (x1 ?? y1)/*T:B<object?>*/.F.ToString();
     }
     static void F2(A<object?>? x2, B<object> y2)
     {
-        (x2 ?? y2)/*T:B<object!>!*/.F.ToString();
+        (x2 ?? y2)/*T:B<object!>*/.F.ToString();
```

Comment:
>! [](start = 30, length = 1)

I don't recognize this syntax (! after a type)

Reply:
Oh, of course it is a comment.

---
In reply to: [185653806](https://github.com/dotnet/roslyn/pull/26528#discussion_r185653806) [](ancestors = 185653806)

Reply:
Yes, there is now an internal `SymbolDisplay` option to display non-nullable reference types with a `!` qualifier. The option is used when displaying types in the debugger and in some tests.

---
In reply to: [185653979](https://github.com/dotnet/roslyn/pull/26528#discussion_r185653979) [](ancestors = 185653979,185653806)